.env

```diff
@@ -51,3 +51,10 @@ MERCURE_JWT_SECRET=!ChangeThisMercureHubJWTSecretKey!
 ###< symfony/mercure-bundle ###
 
 APP_LANGUAGE=pt_BR
+
+###> league/oauth2-server-bundle ###
+OAUTH_PRIVATE_KEY=%kernel.project_dir%/config/jwt/private.pem
+OAUTH_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+OAUTH_PASSPHRASE=6014cc62470302defc5a294bf58656b0
+OAUTH_ENCRYPTION_KEY=42efd1a4afa2d072998915775cba3507
```

Comment:
Is this ok?

Reply:
this is generated automatically when installing the bundle and must be overridden when deploying the app. 

Recipe: https://github.com/symfony/recipes/blob/main/league/oauth2-server-bundle/0.4/manifest.json

Anyway, I'll update it to something like `!PleaseChangeIt!` to make it much clearer.